src/espIdf/unitTest/adapter.ts

```diff
@@ -146,8 +146,7 @@ export class UnitTest {
       TestRunProfileKind.Run,
       runHandler,
       true,
-      undefined,
-      false
+      undefined
```

Comment:
Why is this file modified in this PR ?

Reply:
Reverted this change as it was indeed unnessecary.